Test/sampleModFileDouble2.txt

```diff
@@ -3,5 +3,6 @@ TG   Nxs or Nxt
 PP   Anywhere.
 MM   15.995
 NL   0 or 15
+OM   343
```

Comment:
The masses seen in the histogram, it looks like. How is that different than the MM?